.rhcicd/clowdapp-engine.yaml

```diff
@@ -102,6 +102,23 @@ objects:
             secretKeyRef:
               name: backoffice
               key: token
+        - name: QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE_TYPE
+          value: ${IT_USER_SERVICE_KEY_STORE_FILE_TYPE}
+        - name: QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_FILE
+          valueFrom:
+            secretKeyRef:
+              name: it-services
+              key: keystore.jks
+        - name: QUARKUS_HTTP_SSL_CERTIFICATE_KEY_STORE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: it-services
+              key: it-services-keystorepassword
+        - name: IT_S2S_MP_REST_URL
+          valueFrom:
+            secretKeyRef:
+              name: it-services
+              key: url
```

Comment:
Does the secret exist on both stage and prod?

Reply:
yes, there is one thing to change. but I am also not planning to put it on prod today.
the keys are there, one value is probably not correct. but that's something we can change in vault and not here.